npm/runtime/src/helpers/esm/taggedTemplateLiteralEscape.js

```diff
@@ -0,0 +1,6 @@
+function _taggedTemplateLiteralEscape(e, t) {
+  return t || (t = e.slice(0)), Object.freeze(Object.defineProperty(e, "raw", {
+    value: Object.freeze(t)
+  }));
```

Comment:
esbuild's helper function is like this
```suggestion
  return Object.freeze(Object.defineProperty(e, "raw", {
    value: Object.freeze(t || e.slice())
  }));
```
I think we can use it instead of the babel's one as this would be slightly shorter when minified.

Reply:
Ah, ok. It feels a bit verbose, but it's definitely not a blocker.